tests/unit/getTestResultsByTesterStaffIdFunction.unitTest.ts

```diff
@@ -134,5 +134,81 @@ describe("getTestResultsByTesterStaffId Function", () => {
         expect(result.body).toEqual(JSON.stringify(MESSAGES.BAD_REQUEST));
       });
     });
+    context("with good event", () => {
+      it("should trigger validation and throw 400 error when testerStaffId is null", async () => {
+        const myEvent = {
+          queryStringParameters: {
+            testerStaffId: null,
+            testStationPNumber: "abc123",
+            toDateTime: "01-01-2010",
+            fromDateTime: "01-01-2009",
+            testStatus: "active"
+          }
+        }
+
+        const testResultsMock = jest.fn();
+        TestResultsService.prototype.getTestResultsByTesterStaffId = testResultsMock;
+
+        try {
+          await getTestResultsByTesterStaffId(myEvent);
+        } catch (e) {
+          expect.assertions(4);
```

Comment:
It's worth putting the `expect.assertions(4)` outside of the catch block as in the `await getTestResultsByTesterStaffId(myEvent);` doesn't throw an error, the test will pass. That is if you're always expecting this test to always throw an error.